### DISCUSSION TITLE
How to hide extraneous hints when using `go` extras

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [LazyVim/LazyVim#3305] [Q&A] [answered]
Recently I've started having these type hints in my declarations and function calls. Didn't have them before, I tried disabling them via the `gopls.hints` option but had no luck

```lua
  {
    "neovim/nvim-lspconfig",
    opts = function(_, opts)
      opts.servers.gopls = vim.tbl_deep_extend("force", opts.servers.gopls or {}, {
        buildFlags = { "-tags=manual" },
      })
    end,
  },

```

![screenshot of ide with hints](https://github.com/LazyVim/LazyVim/assets/50072027/0a9734b8-0f58-4f2a-9468-d6083d4c69f7)

**Comment by dpetka2001** [comment, marked as answer]
Either press `<leader>uh` to toggle them off and back on for your current Neovim session or put the following in `~/.config/nvim/lua/plugins/lsp.lua`
```lua
return {
  {
    "neovim/nvim-lspconfig",
    opts = {
      inlay_hints = {
        enabled = false,
      },
    },
  },
}
```
for permanently not showing them when starting Neovim.
